pkgs/by-name/co/coq-lsp/package.nix

```
@@ -0,0 +1,73 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  coq_8_20,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "coq-lsp";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "ejgallego";
+    repo = "coq-lsp";
+    rev = "0.2.3+8.20";
```

```suggestion
    tag = "${finalAttrs.version}+8.20";
```
also, why not package the +9.0 version, since that Coq is also available?